Dockerfile

```diff
@@ -1,15 +1,21 @@
+ARG WORKFLOWS_VERSION=master
+ARG MENDER_ARTIFACT_VERSION=3.7.1
+
 FROM golang:1.18.2-alpine3.15 as builder
 RUN apk add --no-cache \
+    ca-certificates \
     musl-dev \
     gcc \
     git
 WORKDIR /go/src/github.com/mendersoftware/create-artifact-worker
 COPY ./ .
-RUN env CGO_ENABLED=1 go build -o create-artifact
+RUN env CGO_ENABLED=0 go build -o create-artifact
+
+FROM mendersoftware/workflows:$WORKFLOWS_VERSION as workflows
 
-FROM mendersoftware/workflows:master
+FROM alpine:3.15.4
+ARG MENDER_ARTIFACT_VERSION
```

Comment:
Why default value above, and not here?

Reply:
at the top the default value is set, here it is just repeated to be visible in this stage (+ the value is carried over)... if the repeated declaration was missing here, the variable would not be set 👌 (IMO not the most intuitive part of a Dockerfile 🤷‍♂️)